# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pmud/X11R7.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- pmud-0.10/Makefile.vanilla	2008-04-17 12:19:49.000000000 +0200
+++ pmud-0.10/Makefile	2008-04-17 12:19:53.000000000 +0200
@@ -49,7 +49,7 @@
 	install -c snooze /sbin
 	install -c wakebay /sbin
 	install -c fblevel /sbin
-	install -c xmouse /usr/X11R6/bin
+	install -c xmouse /usr/X11/bin
 	install -c Batmon /usr/bin
 	install -c -D pwrctl /etc/power/pwrctl
 	ln -sf /sbin/snooze /usr/bin/apm
